app/controllers/refine/inline/criteria_controller.rb

```diff
@@ -93,12 +93,17 @@ def clear
     handle_filter_update()
   end
 
-  private
+  def merge_groups
+    @criterion = Refine::Inline::Criterion.new(criterion_params.merge(refine_filter: @refine_filter)) 
```

Comment:
Technically we don't need a criterion object to perform the operation required here, but the controller expects a criterion so the action just passes in the first element of the group being acted on. Its not ideal and arguably we should have a conjunction controller or a rethink of how criteria are managed. Today criterion are also acting on their surrounding conjunctions and it can be confusing but I didn't want to mess with this paradigm in this work